controllers/cloud.redhat.com/providers/kafka/strimzi.go

```diff
@@ -658,7 +658,7 @@ func (s *strimziProvider) processTopics(app *crd.ClowdApp) error {
 
 		topicName := fmt.Sprintf("%s-%s-%s", topic.TopicName, s.Env.Name, nn.Namespace)
 		knn := types.NamespacedName{
-			Namespace: getKafkaNamespace(s.Env),
+			Namespace: getKafkaName(s.Env),
```

Comment:
I think this is erroneous, we should be getting the Namespace here, not the name no?